### DISCUSSION TITLE
Question: An explination on quality of RHVoice vs Tacotron/Waveglow

### DISCUSSION THREAD

**Discussion by DonaldTsang** [RHVoice/RHVoice#257] [Q&A] [answered]
Pretext: For someone who really hates unnatural TTS, I am trying to look for FOSS alternatives for Android

RHVoice is a good candidate, however, IDK if this is as natural as Tacotron or Waveglow, or even the others.
- https://github.com/keithito/tacotron
- https://github.com/NVIDIA/waveglow and https://github.com/NVIDIA/flowtron 
- https://github.com/r9y9/gantts and https://github.com/r9y9/nnmnkwii and https://github.com/r9y9/tacotron_pytorch
- https://github.com/soobinseo/Transformer-TTS
- https://github.com/jik876/hifi-gan
- https://github.com/soobinseo/Transformer-TTS

It would be useful to have documentation that helps with the issue.

**Comment by abitrolly** [comment, marked as answer]
I doubt you can compare manually crafted HMM to ones trained with Deep Learning advancements in terms of quality. Maybe the size of HMM could be smaller, but that's needs tests too. The latest Google papers on Tacotron are here https://google.github.io/tacotron/
